hnn_core/extracellular.py

```diff
@@ -301,11 +298,11 @@ def copy(self):
 
     @property
     def times(self):
-        return self._times
+        return np.array(self._times)
 
     @property
     def voltages(self):
-        return self._data
+        return np.array(self._data)
```

Comment:
```suggestion
        return self._voltages
```
Aren't these already cast as arrays elsewhere? Also seems like we could just use `self._voltages` instead of `_data`

Reply:
not really, it's expected to be a list because of these lines:

https://github.com/jonescompneurolab/hnn-core/blob/08daf2024467b46e4efc447efd935645b4fd0aca/hnn_core/parallel_backends.py#L70-L71

I started trying to fix it, but ran into all sorts of issues (zero dimensions concatenation etc) Don't have the patience to fix it now. The PR is already fixing the user facing arrays

Reply:
perhaps @cjayb or you could look into this if you guys are motivated for this ...